### PR TITLE
Opt out of RoslynContentType

### PR DESCRIPTION
**Pull request by mgoertz-msft**
PR for XAML Dev15 RC issue #14565.

XAML needs to opt out of RoslynContentType. There were too many incompatible command handlers that prevented Visual Studio from doing the right thing for XAML.

I also removed a CodeFixProvider that we implemented in VS instead. The other one needs to stay since it references an internal language service.

@CyrusNajmabadi, please review. Thank you!

### CODE REVIEW

File: src/VisualStudio/Xaml/Impl/Diagnostics/XamlDiagnosticIds.cs [hunk 1]
```diff
@@ -4,7 +4,6 @@ namespace Microsoft.CodeAnalysis.Editor.Xaml.Diagnostics
 {
     internal static class XamlDiagnosticIds
     {
-        public const string UnnecessaryNamespacesId = "XAML0001";
-        public const string MissingNamespaceId = "XAML0002";
+        public const string UnnecessaryNamespacesId = "XAML1103";
```
Comment (Pilchie) [inline src/VisualStudio/Xaml/Impl/Diagnostics/XamlDiagnosticIds.cs]: Any reason to re-number the UnnecessaryNamespacesId?

File: src/VisualStudio/Xaml/Impl/Features/InlineRename/XamlEditorInlineRenameService.cs [hunk 2]
```diff
@@ -66,8 +66,6 @@ public InlineRenameInfo(IXamlRenameInfoService renameService, Document document,
             {
                 var references = new List<InlineRenameLocation>();
 
-                references.Add(new InlineRenameLocation(_document, _renameInfo.TriggerSpan));
```
Comment (Pilchie) [inline src/VisualStudio/Xaml/Impl/Features/InlineRename/XamlEditorInlineRenameService.cs]: Why?

**Comment by Pilchie** [comment]
Also tagging @ManishJayaswal for ask mode approval

**Comment by ManishJayaswal** [comment]
Approved for RC pending test run completion.

File: src/VisualStudio/Xaml/Impl/Features/InlineRename/XamlEditorInlineRenameService.cs (hunk 2 above)
Comment (mgoertz-msft) [inline src/VisualStudio/Xaml/Impl/Features/InlineRename/XamlEditorInlineRenameService.cs]: Good question. This is now the job of our service in VS.

**Comment by mgoertz-msft** [comment]
@ManishJayaswal, any ideas what's wrong with the windows_debug_unit64_prtest build?

error MSB3277:  Found conflicts between different versions of the same dependent assembly that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed. [D:\j\workspace\windows_debug---cd6fe12b\src\VisualStudio\VisualStudioDiagnosticsToolWindow\VisualStudioDiagnosticsWindow.csproj] -> 14:16:08.459    89>C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1820,5): 

I didn't touch that project at all. How can I trigger a rebuild for this?

**Comment by Pilchie** [comment]
There is a race in our build that @jaredpar is working on fixing.

Saying "retest windows_debug_unit64_prtest please"

Should trigger a rebuild.

**Comment by Pilchie** [comment]
There is a race in our build that @jaredpar is working on fixing.

Saying retest windows_debug_unit64_prtest please

Should trigger a rebuild.

**Comment by Pilchie** [comment]
Not sure why it isn't.  @333fred?

**Comment by 333fred** [comment]
@Pilchie it's disabled while @mmitche looks into the token depletion. You'll need to hit retry on the Jenkins build page.

**Comment by CyrusNajmabadi** [comment]
Seems ok to me.

**Comment by mgoertz-msft** [comment]
@333fred, where do I hit Retry there? I don't see a possibility to retry anywhere.

**Comment by 333fred** [comment]
@mgoertz-msft, hit details on the build you want to retry, then hit retry on the Jenkins page. Make sure you're logged in, or you won't see the button.

**Comment by mgoertz-msft** [comment]
@333fred, logging in was the missing piece. Thank you!

**Comment by mmitche** [comment]
@333fred @Pilchie That isn't disabled.  Only a few corner cases (like looking back into past comments over a restart.

FYI the trigger phrases I think for you guys got changed so they must be the first thing on a line.

**Pull request merged**